science/paraview/Portfile

```diff
@@ -118,12 +119,23 @@ pre-configure {
         -DCMAKE_C_COMPILER=${configure.cc}
 
        if {[mpi_variant_isset]} {
+           configure.args-delete -DPARAVIEW_USE_MPI:BOOL=OFF
            configure.args-append  \
                -DPARAVIEW_USE_MPI:BOOL=ON \
                -DMPI_C_COMPILER=${mpi.cc} \
                -DMPI_CXX_COMPILER=${mpi.cxx} \
                -DMPIEXEC=${mpi.exec} \
-               -DMPI_Fortran_COMPILER=${mpi.fc}
+               -DMPI_Fortran_COMPILER=${mpi.fc} \
+               -DVTK_USE_SYSTEM_DIY2:BOOL=OFF
+
+            if {${os.major} == 16} {
```

Comment:
Is this *really* only specific to darwin 16, or should it be applied to older OSes as well, in which case the test should be `${os.major} <= 16` ?

Reply:
Hmm, the reports I found were specific to Sierra (and Xcode 9). It could be applicable to older OSes, but I wouldn't know. I suggest we merge and see if we receive problem reports from users.

Reply:
The github report suggest the issue is actually Xcode specific, not OS (not quite the same). If so the check then should actually be based on the Xcode version, and not Darwin.

Reply:
Other github reports talk mainly about Sierra. It also looks like they also apply this solution carte blanche for _all_ Mac OS and Xcode versions. But we can change it to an Xcode-version check if that seems preferable. Can you tell us what that should like like so that we do not have to dig through documentation?

Reply:
you want the xcodeversion PortGroup. a number of ports in the tree use it as an example, e.g. randomly picked https://github.com/macports/macports-ports/blob/master/lang/gcc8/Portfile

Reply:
Ok, I hope I used it correctly :)